docs/modules/ROOT/pages/infra_services/data_plane_manager.adoc

```diff
@@ -325,48 +325,197 @@ http://localhost:8080/goalstate/
-          "id": "f37810eb-7f83-45fa-a4d4-1b31e75399df",
-          "name": "test_cni_port2",
-          "description": "",
-          "network_id": "a87e0f87-a2d9-44ef-9194-9a62f178594e",
-          "tenant_id": null,
-          "admin_state_up": true,
-          "mac_address": null,
-          "veth_name": "veth0",
-          "device_id": null,
-          "device_owner": null,
-          "status": null,
-          "fixed_ips": [
-          "192.168.1.2"
-          ],
-          "allowed_address_pairs": null,
-          "extra_dhcp_opts": null,
-          "security_groups": null,
-          "binding:host_id": "ephost_0",
-          "binding:host_ip": "1.2.3.4",
-          "binding:profile": null,
-          "binding:vnic_type": null,
-          "network_ns": "/var/run/netns/test_netw_ns",
-          "dnsName": null,
-          "dnsAssignment": null,
-          "fast_path": true
-        },
-        "security_group": null
-      }
-    }
-  }
-]
+   "gsinfo":{
+      "vpc":{
+         "project_id":"3dda2801-d675-4688-a63f-dcda8d327f50",
+         "id":"9192a4d4-ffff-4ece-b3f0-8d36e3d88038",
+         "name":"test_vpc",
+         "description":"",
+         "cidr":"192.168.0.0/16"
+      },
+      "subnets":[
+         {
+            "project_id":"3dda2801-d675-4688-a63f-dcda8d327f50",
+            "vpc_id":"9192a4d4-ffff-4ece-b3f0-8d36e3d88038",
+            "id":"a87e0f87-a2d9-44ef-9194-9a62f178594e",
+            "name":"test_subnet",
+            "description":"",
+            "cidr":"192.168.1.0/20",
+            "gateway_ip":"192.168.1.5",
+            "availability_zone":"uswest-1",
+            "dhcp_enable":false,
+            "primary_dns":null,
+            "secondary_dns":null,
+            "dns_list":null
+         },
+         {
+            "project_id":"3dda2801-d675-4688-a63f-dcda8d327f50",
+            "vpc_id":"9192a4d4-ffff-4ece-b3f0-8d36e3d88038",
+            "id":"a87e0f87-a2d9-44ef-9194-9a62f178594f",
+            "name":"test_subnet",
+            "description":"",
+            "cidr":"192.168.2.0/20",
+            "gateway_ip":"192.168.2.5",
+            "availability_zone":"uswest-1",
+            "dhcp_enable":false,
+            "primary_dns":null,
+            "secondary_dns":null,
+            "dns_list":null
+         }
+      ],
+      "ports":[
+         {
+            "project_id":"3dda2801-d675-4688-a63f-dcda8d327f50",
+            "id":"f37810eb-7f83-45fa-a4d4-1b31e75399df",
+            "name":"test_cni_port2",
+            "description":"",
+            "network_id":"a87e0f87-a2d9-44ef-9194-9a62f178594e",
+            "tenant_id":null,
+            "admin_state_up":true,
+            "mac_address":null,
+            "veth_name":"veth0",
+            "device_id":null,
+            "device_owner":null,
+            "status":null,
+            "fixed_ips":[
+               {
+                  "ipAddress":"192.168.1.2",
+                  "subnetId":"a87e0f87-a2d9-44ef-9194-9a62f178594f"
```

Comment:
Use the first subnet id *9a62f178594e as 192.168.1.2 falls into the ip range of the first subnet. Please check the other ports as well.

Reply:
changed in 8da6742